src/LanguageFunctions.ts

```diff
@@ -210,16 +214,29 @@ export async function RefreshXlfFilesFromGXlf(sortOnly?: boolean): Promise<{
                                 NumberOfUpdatedSources++;
                                 sourceIsUpdated = true;
                             }
-                            if (!langTargetElement) {
+                            let recreateTarget = false;
```

Comment:
* Move declaration to top of the function?
* Would this be better for readability?
```TypeScript
recreateTarget = langTargetElement && (langTargetElement.textContent === GetNotTranslatedToken())
if (recreateTarget) {
...
}
```